File: lab7/count_word.pl
#!/usr/bin/perl -w

$#ARGV == 0 or die $!;
$count = 0;

while($line = <STDIN>){
	@wc = $line =~ /\W$ARGV[0]\W|^$ARGV[0]\W/gi;
    $count += @wc;	
}

print "$ARGV[0] occured $count times\n";

File: lab7/count_word.py
#!/usr/bin/python

import sys;
import re;

if(len(sys.argv) != 2):
	sys.exit("Usage:");

count = 0;
for line in sys.stdin:
	words = re.split("\W",line);
	for word in words:
		if(word == sys.argv[1]):
			count += 1;
print sys.argv[1],"ocurred",count,"times"; 
